cypress/e2e/navigation.cy.ts

```diff
@@ -72,4 +72,36 @@ describe('Navigation Test', () => {
     profilePage.verifySchoolDetails(schoolDetails);
     cy.logout();
   });
+
+  it('Should navigate to enrolled contests and verify filter', () => {
+    const userLoginOptions = loginPage.registerMultipleUsers(1);
+    cy.login(userLoginOptions[0]);
+
+    cy.get('[data-nav-user]').should('be.visible');
+    cy.get('[data-nav-user]').click();
+    cy.get('[data-nav-user-contests]').should('be.visible');
+
+    cy.get('[data-nav-user-contests]').first().click();
+
+    cy.waitUntil(() =>
+      cy
+        .url()
+        .should(
+          'include',
+          '/arena/?page=1&tab_name=current&sort_order=none&filter=signedup',
+        ),
+    );
+
+    cy.get('button.dropdown-toggle-no-caret').eq(1).click();
```

Comment:
Acá también puedes evitar el uso del selector posicional. Parece que el selector que estás usando es muy genérico. 

Así que la solución puede ser agregar `data-attributes` en los dos dropdowns, el primero puede llamarse `data-dropdown-order` y el que te interesa en esta prueba pudiera llamarse `data-dropdown-filter`.

De esta forma, acá cambiarías tu selector por algo así:
```suggestion
    cy.get('[data-dropdown-filter]>button').click();
```